src/exosite.c

```diff
@@ -376,8 +426,20 @@ exo_state exo_operate(exo_op *op, uint8_t count)
       break;
     }
   }
+}
+
+// process all ops that are in an active state
+static void exo_process_active_ops(exo_op *op, uint8_t count)
+{
+  uint8_t buf[576];
```

Comment:
This actually brings up a good point. That number is from RFC791: "All hosts must be prepared to accept datagrams of up to 576 octets (whether they arrive whole or in fragments)."

I would like to use dynamically sized, stack-allocated buffers so that it's only limited by the amount of ram we have left, but that requires C99. Is that too strict of a limitation for embedded code that will be used on a compile that is who knows how old?

For now I'll update it to give it a name an a reference to the RFC.